### PR TITLE
add custom check notebook

### PR DESCRIPTION
**Pull request by Hrithikd**

### CODE REVIEW

File: examples/qa_eval_custom_check.ipynb [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    "\n",
+    "# Import Okareo's handler for OpenAI models\n",
+    "from okareo.model_under_test import OpenAIModel\n",
+    "\n",
+    "# Create an instance of the OpenAI client\n",
+    "client = OpenAI(api_key=OPENAI_API_KEY)\n",
+    "\n",
+    "# Define a function to call the OpenAI API\n",
+    "# This function will be used to query the specified OpenAI model\n",
+    "def get_turbo_summary(messages, model=\"gpt-3.5-turbo\", temperature=0, max_tokens=500):\n",
```
Comment (okasignal) [inline examples/qa_eval_custom_check.ipynb:114]: is get_turbo_summary being used?

File: examples/qa_eval_custom_check.ipynb [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from okareo.checks import ModelBasedCheck, CheckType\n",
+    "check = okareo.create_or_update_check(\n",
+    "    name=\"Relevance Check\",\n",
+    "    description=\"Relevance check for QA with context\",\n",
+    "    check=ModelBasedCheck(\n",
+    "        prompt_template=\"\"\"Rate the relevance of the answer on a scale of 1 to 5. The output should be only 1 number.\n",
```
Comment (okasignal) [inline examples/qa_eval_custom_check.ipynb:163]: Can we start from g-eval relevance prompt and modify it with -summary +question?

**Review by okasignal — COMMENTED** [review]
Is there a way to further simplify/remove redundancy and also create better comments similar to structure Stefan has created for other notebooks?

**Review by mdelrosa — APPROVED** [review]
Note needed for this commit, but would love to see an example of a pass_fail check as well. I think SnapOn cares about score-based checks, so it's probably fine to ship this as is.

**Pull request merged**